recirq/optimize/mpg.py

```diff
@@ -67,56 +74,31 @@ def clip_by_norm(t, clip_norm):
     return t * clip_norm / np.maximum(l2_norm, clip_norm)
 
 
-class ConstantSchedule(object):
```

Comment:
Why did you remove `ConstantSchedule`?

Reply:
oh, because it is not used in the code? Shall I put it back?

Reply:
Well it was there before so I was wondering why you removed it :) It's up to you and what use cases you want to support

Reply:
ha, I find the schedule is not exposed to the "outside". so, previous constant schedule can not be used. 
But, in practice, it always good to use the decay schedule. If anyone finds the constant schedule is better, I can then put it back.